### DISCUSSION TITLE
Adding math symbols?

### DISCUSSION THREAD

**Discussion by dbleyl** [NicolasH2/ggbrace#7] [Q&A] [answered]
How do we include math symbols in a label?

I have tried direct plotmath syntax as well as `latex2exp` which outputs plotmath.  In the example below, "test" renders correctly.

```r
 geom_brace( aes(c(1,20), c(9,8.5), label="test"), labelsize=4, rotate=180, alpha=0.5, inherit.data=F)
```

![image](https://user-images.githubusercontent.com/218076/215900335-ebdc040c-c393-434b-8930-0faa67d1e3a9.png)

`geom_brace( aes(c(1,20), c(9,8.5), label=as.expression(bquote(mu == .(first_mu)))), labelsize=4, rotate=180, alpha=0.5, inherit.data=F)`

Caused by error in `compute_aesthetics()`:
! Aesthetics are not valid data columns.
✖ The following aesthetics are invalid:
✖ `label = as.expression(bquote(mu == .(first_mu)))`
ℹ Did you mistype the name of a data column or forget to add `after_stat()`?

I can render something like this:
![image](https://user-images.githubusercontent.com/218076/215903064-7f5073db-5383-4c06-9ea8-187a6efdf546.png)

```R
geom_brace( aes(c(1,20), c(9,8.5), label="test"), labelsize=4, rotate=180, alpha=0.5, inherit.data=F) +
  geom_label(x=10.5, y=8.0, label=as.expression(bquote(mu == .(first_mu))))
```
Using a separate label, but positioning and the extra step seem suboptimal.

**Comment by NicolasH2** [comment, marked as answer]
Hi @dbleyl ,

if we are talking about unicode symbols, then this worked for me:

```r
library(ggplot2)
library(ggbrace)

plt <- ggplot(iris, aes(x=Sepal.Length, y=Sepal.Width, color=Species, label=Species)) + 
  geom_point() +
  theme_classic() +
  theme(legend.position="none")

plt + geom_brace(aes(c(4,7), c(4, 4.5), label="\u03b1 \u03a9 \u00be \u026E"), inherit.data=F, labelsize=5)
```

you can find a list of all the unicodes in the following link. Just replace the `U+` in the beginning of each unicode with `\u`: [https://en.wikipedia.org/wiki/List_of_Unicode_characters](https://en.wikipedia.org/wiki/List_of_Unicode_characters)
